--- a/src/snr/screen/content_pages.py	2020-06-22 13:20:52.000000000 +0200
+++ b/src/snr/screen/content_pages.py	2024-07-25 22:48:08.404442203 +0200
@@ -2,7 +2,7 @@
 
 import curses
 from textwrap import wrap
-from textwrap2 import wrap as wrap2
+from textwrap3 import wrap as wrap3
 from .pages import Pages
 
 class ContentPages(Pages):
@@ -62,7 +62,7 @@
             for index, paragraph in enumerate(content):
                 if self.book.has_dict() and self.hyphenation:
                     try:
-                        lines_of_text = wrap2(paragraph, self.page_columns, use_hyphenator=self.book.get_lang_dict())
+                        lines_of_text = wrap3(paragraph, self.page_columns, use_hyphenator=self.book.get_lang_dict())
                     except:
                         lines_of_text = wrap(paragraph, self.page_columns)
                 else:
@@ -92,7 +92,7 @@
             content = self.book.get_chapter_title(self.chapter)
             if self.book.has_dict() and self.hyphenation:
                 try:
-                    lines_of_text = wrap2(content, self.page_columns, use_hyphenator=self.book.get_lang_dict())
+                    lines_of_text = wrap3(content, self.page_columns, use_hyphenator=self.book.get_lang_dict())
                 except:
                     lines_of_text = wrap(content, self.page_columns)
             else:
